### PR TITLE
parser: fix obscure error message in `bit(65)` column type

### PR DESCRIPTION
**Pull request by tangenta**
### What problem does this PR solve? 

Fix pingcap/tidb#11923.

### What is changed and how it works?
Remove the check against field-display width of `BIT` .

The template error message is 
```
ErrTooBigDisplaywidth: "Display width out of range for column '%-.192s' (max = %d)",
```
It requires a column name argument, which is impossible to be accessed in the semantic action of `NumericType`. 

Although we can move the validation code to the two-level-higher `ColumnDef`, it can break the encapsulation. So let TiDB handle it.

Related test: pingcap/tidb#11985

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: parser.y [hunk 1]
```diff
@@ -2137,7 +2137,7 @@ ColumnDef:
 			yylex.AppendError(yylex.Errorf("Invalid column definition"))
 			return 1
 		}
-        $$ = colDef
+        	$$ = colDef
```
Comment (zz-jason) [inline parser.y]: could you replace all the spaces to tab?

Comment (tangenta) [reply parser.y]: Done.

**Comment by kennytm** [comment]
> So let TiDB handle it.

Is there a corresponding TiDB PR to reject `BIT(65)`?

**Comment by tangenta** [comment]
@kennytm I have found the related code in TiDB. And I am trying to test it locally.

**Comment by leoppro** [comment]
LGTM

**Pull request merged**